integration/api/POSTGRES_PASSWORD

```diff
@@ -0,0 +1 @@
+passw0rd
```

Comment:
I would just put it in yml compose and not as secret file for now for simplicity